cypress/integration/components/generator.spec.js

```diff
@@ -36,8 +28,6 @@ describe('Generator component', () => {
             // time to 🦆 type...
             expect(win.pdfWorker).respondTo('postMessage');
             expect(win.pdfWorker).respondTo('terminate');
-            expect(win.pdfWorker).respondTo('onmessage');
-            expect(win.pdfWorker).respondTo('onerror');
```

Comment:
These are undefined now, because instead of setting these, we add event listeners to the `Worker`, so the test would fail here, even though the Worker has been correctly set up.